libraries/Microsoft.Bot.Builder.Ai.LUIS/LuisRecognizer.cs

```diff
@@ -84,8 +84,8 @@ private static string NormalizedIntent(string intent)
         private static JObject GetIntents(LuisResult luisResult)
         {
             return luisResult.Intents != null ?
-                JObject.FromObject(luisResult.Intents.ToDictionary(i => NormalizedIntent(i.Intent), i => i.Score ?? 0)) :
-                new JObject { [NormalizedIntent(luisResult.TopScoringIntent.Intent)] = luisResult.TopScoringIntent.Score ?? 0 };
+                JObject.FromObject(luisResult.Intents.ToDictionary(i => NormalizedIntent(i.Intent), i => new JObject(new JProperty("score", i.Score ?? 0)))) :
```

Comment:
Strongly consider breaking this line up into multiple lines. 

As it sits, it's complex and hard to debug.

Reply:
Done